plugins/core/gov.nasa.jpl.imce.oml.viewpoint/src/gov/nasa/jpl/imce/oml/viewpoint/TerminologyService.xtend

```diff
@@ -0,0 +1,337 @@
+package gov.nasa.jpl.imce.oml.viewpoint

```

Comment:
Please add the copyright header.

`/*
 * Copyright 2017 California Institute of Technology ("Caltech").
 * U.S. Government sponsorship acknowledged.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * License Terms
 */`